### ISSUE TITLE
Reduce the log severity for "Callable request verification passed"

### ISSUE THREAD

**Issue by digibake** [firebase/firebase-functions#952]
I wonder if you could consider reducing the severity of the 'Callable request verification passed' log message from INFO to DEBUG in file:

```
581: logger.info('Callable request verification passed', logPayload);
```

https://github.com/firebase/firebase-functions/blob/fdcd3090b4b139f52c2baea63b33a46c1ca0c992/src/common/providers/https.ts

While reading through the logs in log explorer there are thousands of "Callable request verification passed" log messages and it would be so much easier to read other log messages from the functions if these were at a reduced level of severity and could be filtered out.

Feel free to close this issue if it's not relevant, or maybe add the "NOTICE" level of severity to the logger in file:
https://github.com/firebase/firebase-functions/blob/fdcd3090b4b139f52c2baea63b33a46c1ca0c992/src/logger/index.ts
so that we could raise the priority for the log messages that we're looking out? Everything gets dumped into INFO at the moment.

### Related issues

Possibly related to: https://github.com/firebase/firebase-functions/issues/903

### [REQUIRED] Version info

**node:**
v14.17.0

**firebase-functions:**
3.15.3

**firebase-tools:**
9.16.0

**firebase-admin:**
9.11.0

### [REQUIRED] Test case

* Assuming a callable function is set up, run the following query in Logs Explorer on Google Cloud:
```
severity=INFO
resource.type="cloud_function"
```

### [REQUIRED] Steps to reproduce

* Create a Callable function
* Run the Callable function
* Open the logs in Logs Explorer on Google Cloud
* Run the query:
```
severity=INFO
resource.type="cloud_function"
```

### [REQUIRED] Expected behavior

```
Debug 2021-08-16 09:01:19.186 BST Function_name ID  Callable request verification passed
```

### [REQUIRED] Actual behavior

```
Info 2021-08-16 09:01:19.186 BST function_name ID  Callable request verification passed
```

### Were you able to successfully deploy your functions?

Yes, functions run fine. It's a minor thing that would make my life easier ;-)

**Comment by inlined** [comment]
We agree with your suggestion and wish we had released this way all along. Changing the log severity may be a breaking change however so we're going to wait until the next major version to make the change.

**Comment by digibake** [comment]
Great! Thanks, looking forward to it. Well done guys, keep up the good work.

**Comment by taeold** [comment]
This was fixed in Firebase Functions v4.0.0
